<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Data Vault Explorer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/nSOTScannerResources.qrc">
    <normaloff>:/nSOTScanner/Pictures/jj_schema.svg</normaloff>:/nSOTScanner/Pictures/jj_schema.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
	background-color: rgb(0, 0, 0);}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,7,1,1,0,1" columnstretch="1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1" columnminimumwidth="1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="Directories">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#838383;&quot;&gt;Directories:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8" colspan="4">
     <widget class="QLabel" name="DataSets">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#838383;&quot;&gt;Data Sets:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="14" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;File&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="8">
     <widget class="QListWidget" name="dirList">
      <property name="styleSheet">
       <string notr="true">QListWidget#dirList{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}


QScrollBar:horizontal {
   border: 0px;
   background: black;
   height: 15px;
}
QScrollBar:vertical {
   border: 0px;
   background: black;
   width: 15px;
}</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8" colspan="8">
     <widget class="QListWidget" name="fileList">
      <property name="styleSheet">
       <string notr="true">#fileList{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}


QScrollBar:horizontal {
   border: 0px;
   background: black;
   height: 15px;
}
QScrollBar:vertical {
   border: 0px;
   background: black;
   width: 15px;
}</string>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;Write to Directory:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="8" colspan="6">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;Open Data Set:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="8">
     <widget class="QLineEdit" name="currentDir">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#currentDir{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="8" colspan="8">
     <widget class="QLineEdit" name="currentFile">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#currentFile{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="horizontalSpacer_24">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>23</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1">
     <spacer name="horizontalSpacer_18">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>22</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="2">
     <spacer name="horizontalSpacer_22">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>23</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="3">
     <spacer name="horizontalSpacer_20">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="4">
     <spacer name="horizontalSpacer_23">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="5">
     <spacer name="horizontalSpacer_17">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="6">
     <spacer name="horizontalSpacer_21">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="7">
     <spacer name="horizontalSpacer_19">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="8">
     <spacer name="horizontalSpacer_16">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="9">
     <spacer name="horizontalSpacer_15">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>22</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="10">
     <spacer name="horizontalSpacer_12">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="11">
     <spacer name="horizontalSpacer_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="12">
     <spacer name="horizontalSpacer_14">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="13">
     <spacer name="horizontalSpacer_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="14">
     <spacer name="horizontalSpacer_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="15">
     <spacer name="horizontalSpacer_13">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="addDir">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Add Directory</string>
      </property>
      <property name="styleSheet">
       <string notr="true">#addDir{
image:url(:/nSOTScanner/Pictures/addPlotIcon.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#addDir{
image:url(:/nSOTScanner/Pictures/addPlotIcon.png);
background: black;
border: 2px solid white;
border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="home">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Go to Root</string>
      </property>
      <property name="styleSheet">
       <string notr="true">#home{
image:url(:/nSOTScanner/Pictures/homeIcon.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#home{
image:url(:/nSOTScanner/Pictures/homeIcon.png);
background: black;
border: 2px solid white;
border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="back">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Back</string>
      </property>
      <property name="styleSheet">
       <string notr="true">#back{
image:url(:/nSOTScanner/Pictures/backIcon.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#back{
image:url(:/nSOTScanner/Pictures/backIcon.png);
background: black;
border: 2px solid white;
border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_dvexplorer_refresh">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Refresh</string>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_dvexplorer_refresh{
image:url(:/nSOTScanner/Pictures/refreshIcon.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#refresh{
image:url(:/nSOTScanner/Pictures/refreshIcon.png);
background: black;
border: 2px solid white;
border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="10" colspan="4">
     <widget class="QPushButton" name="cancelSelect">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed#cancelSelect{
color: rgb(0,0,0);
background-color:rgb(131,131,131);
border: 1px solid rgb(0,0,0);
border-radius: 5px
}

QPushButton#cancelSelect{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="4">
     <widget class="QPushButton" name="select">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed#select{
color: rgb(0,0,0);
background-color:rgb(131,131,131);
border: 1px solid rgb(0,0,0);
border-radius: 5px
}

QPushButton#select{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLabel" name="dirName">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;Root&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/nSOTScannerResources.qrc"/>
 </resources>
 <connections/>
</ui>
